api/v1alpha1/status.go

```diff
@@ -104,6 +104,9 @@ type RuntimeStatus struct {
 	// +optional
 	FuseNumberAvailable int32 `json:"fuseNumberAvailable,omitempty"`
 
+	// The setup time of runtime in seconds
+	SetupPeriodSeconds *int64 `json:"setupPeriodSeconds,omitempty"`
```

Comment:
I think it's better to provide readable duration. You can learn from `duration` time of K8s implementation.

Reply:
done